catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/EventProcessorImpl.java

```diff
@@ -435,8 +435,4 @@ public DeleteResponse process(DeleteResponse deleteResponse) throws PluginExecut
         return deleteResponse;
     }
 
-    public static enum DateType {
```

Comment:
This could break third party bundles. We are exporting an impl which nowadays we don't do (hence all the embedding of catalog-core-api-impl). @pklinef might give us the OK though.